lnwire/lnwire.go

```diff
@@ -76,6 +76,9 @@ func (a addressType) AddrLen() uint16 {
 
 // WriteElement is a one-stop shop to write the big endian representation of
 // any element which is to be serialized for the wire protocol.
+//
+// TODO(yy): rm this method once we finish dereferencing it from other
+// packages.
```

Comment:
One thing that would be useful here (still getting through the commits) to make sure we're not breaking anything in the process here would be to retain this method (rename it likely), then use [`quick.CheckEqual`](https://golang.org/pkg/testing/quick/#CheckEqual) to ensure that the encoding produced here doesn't deviate with any input. Eventually (post Go 1.7) we'll be able to also use the fuzzing in the standard library to implement differential fuzzing.

Reply:
Sure, I think atm only package `chanbackup` is using it, will check!